cmd/keysync/keysync.go

```diff
@@ -72,7 +74,7 @@ func main() {
 	}
 
 	if config.SentryDSN != "" {
-		hook, err := configureLogrusSentry(config.SentryDSN, config.CaFile)
+		hook, err := configureLogrusSentry(config.SentryDSN, config.SentryCaFile)
```

Comment:
nit: I think Go prefers to capitalize stuff like `CA` (similar to how you have `SentryDSN`), but since we have stuff like `CaFile` all over keysync, we can address that some other time.